pkg/nemesis/iochaos.go

```diff
@@ -161,18 +206,19 @@ func (m ioMixed) template(
 		Delay:      args[0],
 		Errno:      args[1],
 		Percent:    args[2],
+		Methods:    methods,
 	}
 }
 
 func (m ioMixed) defaultTemplate(ns, configName string, pods []string) chaosv1alpha1.IoChaosSpec {
 	return m.template(ns, pods, chaosv1alpha1.OnePodMode,
-		chaosv1alpha1.FileSystemLayer, configName, "50ms", randomErrno, "50")
+		chaosv1alpha1.FileSystemLayer, configName, nil, "50ms", randomErrno, "50")
 }
 
 type ioChaos interface {
 	ioChaosType() chaosv1alpha1.IOChaosAction
 	template(ns string, pods []string, podMode chaosv1alpha1.PodMode,
-		layer chaosv1alpha1.IOLayer, configMap string, args ...string) chaosv1alpha1.IoChaosSpec
+		layer chaosv1alpha1.IOLayer, configMap string,methods []string, args ...string) chaosv1alpha1.IoChaosSpec
```

Comment:
A small nit. It is better to add a space between `configMap string,` and `methods []string`

Reply:
ok, i'll modify it.